modules/ROOT/pages/notifications/gqlstatus-info-codes.adoc

```diff
@@ -0,0 +1,51 @@
+:description: The page contains GQLSTATUS info codes producing results such as SUCCESS, NO DATA, and OMITTED RESULT.
+
+[[gql-status-info-codes]]
+= Primary GQLSTATUS codes
```

Comment:
```suggestion
= General GQLSTATUS codes
```

Reply:
or it could be: GQL statuses for the `SUCCESS`, `OMITTED RESULT`, and `NO DATA` results.
But it's a long title

Reply:
Actually, in CIP-105, they are called primary.

Reply:
_"The primary GQL-status object is returned separately and accompanied with a list only constraining additional GQL-status objects."_

Reply:
That is not referring to the same concept though, I think. According to GQL if we e.g. run a successful query with a deprecation we might get 01N01 (the deprecation) and 00000 (success). In this case it is 01N01 which is the primary GQLSTATUS because it is the most "important" or "severe" one. Implementation wise, we always return a single list with all of the sorted in an order so the primary object is first. In case GQL mandates it we will add method to return the primary object separately, where the implementation is pretty much just "take the first element in the list". 

So to call this chapter Primary GQLSTATUS would not be correct. I think General sounds OK. Does this make sense @renetapopova @NataliaIvakina @boggle ?